packages/react-widgets/src/hooks/useWidgetFetch.js

```diff
@@ -93,6 +107,36 @@ export default function useWidgetFetch(
     [global, viewport, spatialFilter]
   );
 
+  const source2 = useMemo(() => {
```

Comment:
Probably a more descriptive name for this parameter would help our future ourselves? What about `enrichedSource`?